osu.Game/Online/API/Requests/Responses/SoloScoreInfo.cs

```diff
@@ -180,7 +180,7 @@ public ScoreInfo ToScoreInfo(RulesetStore rulesets, BeatmapInfo? beatmap = null)
         {
             Rank = score.Rank,
             TotalScore = (int)score.TotalScore,
-            Accuracy = score.Accuracy,
+            Accuracy = Math.Round(score.Accuracy, 4, MidpointRounding.ToEven),
```

Comment:
I believe this will cause values like `99.9999%` to round up to `100%`, which is frowned upon as far as https://github.com/ppy/osu/pull/12181 is concerned (in fact, that exact PR is the reason why accuracies are no longer rounded).

Reply:
good call, will revisit this